src/components/L2/Dropdowns/FeeDropdown.tsx

```diff
@@ -17,7 +17,8 @@ export interface GasPriceData {
   suggestedBaseFeePerGas: number;
 }
 
-export const formatWait = (wait: number) => Math.round(wait * 100) / 100;
+// Convert seconds to minutes prettily
+export const formatWait = (wait: number) => Math.round((wait * 100) / 60) / 100;
```

Comment:
was this used anywhere else?

Reply:
Doesn't look like it, not sure why it lives in the component. Will move it